package-list.json

```diff
@@ -6,24 +6,26 @@
     "list": [
       {
         "version": "current",
-        "descmd": "Continuous Integration Build (latest in version control)",
-        "path": "https://build.fhir.org/ig/HL7/fhircast-docs",
+        "path": "'http://hl7.org/fhir/uv/fhircast-docs",
+        "desc": "Continuous Integration Build (latest in version control)",
+        "sequence": "STU3",
+        "fhirversion": "4.0.1",
         "status": "ci-build",
-        "current" : true
-      },
+        "current": true,
+        "date": "2022-03-24"
+        },
       {
         "version": "1.2.0-ballot",
         "desc": "Ballot for STU Release 3",
         "status": "ballot",
         "sequence": "STU3",
         "fhirversion": "4.0.1",
-        "path": "http://hl7.org/fhir/uv/fhircast/2022May",
-        "date": "2022-05-01",
-        "current" : true
+        "path": "http://hl7.org/fhir/uv/fhircast-docs/2022May",
```

Comment:
Lynn said that the url to the eventual spec on hl7.org will not contain "-docs". I worry this is re-introducing a problem.